src/components/application_manager/rpc_plugins/vehicle_info_plugin/test/CMakeLists.txt

```diff
@@ -64,7 +64,7 @@ set(LIBRARIES
   HMI_API
   MOBILE_API
   connectionHandler
-  vehicle_info_plugin
```

Comment:
@ychernysheva why static?

Reply:
Because we need here vehicle_info_plugin library without linkage to "real" Message Helper class. Same trick was done before for SDL RPC plugin.